contrib/get-github-actions-status.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/env python3
+
+import json
+import sys
+import urllib.parse
+import urllib.request
+from time import sleep
+
+
+def get_all_runs():
+    url = "https://api.github.com/repos/syslog-ng/syslog-ng/actions/runs?branch=master"
+    request = urllib.request.Request(url, None, {})
+    with urllib.request.urlopen(request) as response:
+        binary_response = response.read()
+
+    return json.loads(binary_response.decode())["workflow_runs"]
+
+
+def main():
+    workflow = sys.argv[1]
+    while True:
+        runs = filter(lambda run: run["name"] == workflow, get_all_runs())
+        latest_run = max(runs, key=lambda run: run["updated_at"])
```

Comment:
minor improvement idea for input validation: if the `workflow` is not found in the list of github workflows, then the call to `max()` returns a ValueError. 

`get_all_runs` could have some error handling or an input validation after the `filter()`

Reply:
note: I tried to simplify it further by using the "per_page" attribute in the query. My tests shows that it will return the latest run, which will simplify the code, and reduce the amount of transferred data. However there is no mention of any ordering in the API documentation :( So no luck there.

Otherwise LGTM!

Reply:
@gaborznagy I added the necessary checks, which happened to be different after changing the API, but I think they serve the same purpose, as you mentioned.

@szemere Yes, I also had the same observation as you, but no definite thing in the documentation. I prefer to keep the `max` logic.